.github/workflows/build-deploy-k8s-test-azure.yml

```diff
@@ -38,6 +39,16 @@ jobs:
           container-registry-password: ${{ secrets.REGISTRY_PASSWORD }}
           secret-name: alkemio-server-secret
 
+      - uses: azure/setup-kubectl@v3
+          with:
+            version: "v1.22.0" # default is latest stable, fixing it to a compatible version
+          id: install
+
+      - name: Migrate database
```

Comment:
Split the jobs, have `build`, `migrate`, `deploy` as separate jobs. 
In the `migrate` you will need to have an additional action - `Azure/k8s-deploy@v1` to deploy the manifest with the sha, similar to what we have for the server. You will need to add a manifest for the `cronJob` to the repo - look at `25-server-deployment-dev.yaml` as reference.